rules/phpunit/tests/Rector/MethodCall/CreateMockToCreateStubRector/Fixture/skip_sonata_expects.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Rector\phpunit\Tests\Rector\MethodCall\CreateMockToCreateStubRector\Fixture;
+
+use PHPUnit\Framework\TestCase;
+
+class SkipSonataExpects extends TestCase
+{
+    public function testItBehavesAsExpected(): void
+    {
+        $datagrid = $this->createMock(DatagridInterface::class);
+
+        $datagrid->expects($this->once())
+            ->method('getPager')
+            ->willReturn($pager);
+    }
+}
```

Comment:
@greg0ire I've just added test case for:
https://github.com/rectorphp/rector/issues/3157

And it is skipped.

If the problem still remains, we need either failing test case (like this fixture, just failing) or failing demo. We're unable to fix that without any replicable proof.